types/config-store.d.ts

```diff
@@ -4,58 +4,6 @@ declare module "fastly:config-store" {
    * Class for accessing [Fastly Edge Dictionaries](https://docs.fastly.com/en/guides/about-edge-dictionaries).
    *
    * **Note**: Can only be used when processing requests, not during build-time initialization.
-   * 
-   * @example
-   * <script async defer src="https://fiddle.fastly.dev/embed.js"></script>
-   * In this example we have an Edge Dictionary named 'animals' and we return the 'cat'
-   * entry as the response body to the client.
-   * 
-   * <script type="application/json+fiddle">
```

Comment:
This big example comment is not useful in IDE's -- this is a legacy example comment from when our documentation website was generated from the type definitions